akka-actor/src/main/scala/akka/event/ActorClassificationUnsubscriber.scala

```diff
@@ -15,7 +15,7 @@ import akka.util.unused
  *
  * Watches all actors which subscribe on the given event stream, and unsubscribes them from it when they are Terminated.
  */
-protected[akka] class ActorClassificationUnsubscriber(bus: ManagedActorClassification, debug: Boolean)
```

Comment:
@smarter first of all thanks for all your help.
This is the repro I promised to provide:
  - revert this change (here and in EventBus the compiler will tell you the precise line)
  - run `sbt -Dakka.build.scalaVersion=3.0` and `akka-actor/compile`
 The compiler will crash saying that a class file is missing, any attempt to turn it around failed for me and, since it's internal API decided to exclude the mima check (which I think is fair).
 
 I'm sorry, but when I attempted to reduce it I failed a couple of times ...

Reply:
I don't personally have time to look into this right now, can you open an issue against dotty with reproduction steps?

Reply:
done here:
https://github.com/lampepfl/dotty/issues/11226
thanks again for the support!